src/attrs.rs

```diff
@@ -86,16 +86,25 @@ impl quote::ToTokens for Attrs {
     }
 }
 
+const MERMAID_INIT_SCRIPT: &str = r#"
+    var docs_theme = currentTheme.href;
+    var amrn_dark = docs_theme.endsWith("dark.css") || docs_theme.endsWith("ayu.css") 
+    var amrn_theme = amrn_dark ? 'dark' : 'default';
+    mermaid.initialize({'startOnLoad':'true', 'theme': amrn_theme });    
+"#;
+
 fn generate_diagram_rustdoc<'a>(parts: impl Iterator<Item = &'a str>) -> TokenStream {
     let preamble = iter::once(r#"<div class="mermaid">"#);
     let postamble = iter::once("</div>");
 
     let mermaid_js_include = format!(r#"<script src="{}" crossorigin="anonymous"></script>"#, MERMAID_JS);
+    let mermaid_js_init = format!(r#"<script>{}</script>"#, MERMAID_INIT_SCRIPT);
```

Comment:
Why not embed `<script>`s into `MERMAID_INIT_SCRIPT` directly?

Reply:
It seems rather strange to go trough formatting on every diagram when all the parts are known beforehand.

Reply:
There's formatting anyway, in quote! -- this one allocation doesn't affect anything.

Reply:
Okay